src/Twig/UrlBuilderExtension.php

```diff
@@ -34,12 +36,16 @@ public function __construct(Builder $builder)
     public function getFilters()
     {
         return [
-            new \Twig_SimpleFilter('imageUrl', [$this, 'imageUrl']),
+            new TwigFilter('imageUrl', [$this, 'imageUrl']),
         ];
     }
 
     /**
-     * buildUrl
```

Comment:
Useless as it is already defined in PHP directly.

Is it a phpstorm issue or something like that ?

Reply:
I removed it because it was not current method name. I wrote it back with `imageUrl`